motion_config.html

```diff
@@ -5185,6 +5190,66 @@ <h3><a name="track_stepsize"></a> track_stepsize </h3>
         Number of steps to make.
         Only used for stepper motor tracking.
         <p></p>
+        
+        <p></p>
+
+        <h3><a name="track_generic_move"></a> track_generic_move </h3>
+        <p></p>
+        <ul>
+          <li> Type: String</li>
+          <li> Range / Valid values: Max 4095 characters</li>
+          <li> Default: </li>
+        </ul>
+        <p></p>
+        The full path and file name of the command to execute to move a PTZ camera in generic tracking mode
+        <p></p>
+        Several parameters are provided as environment variables:
+        <ul>
+          <li> When called to recenter the camera: TRACK_ACTION=center TRACK_XOFF TRACK_YOFF</li>
+          <li> When called to track a motion: TRACK_ACTION=move TRACK_CENT_X TRACK_CENT_Y TRACK_CENT_WIDTH TRACK_CENT_HEIGHT TRACK_CENT_MINX TRACK_CENT_MAXX TRACK_CENT_MINY TRACK_CENT_MAXY TRACK_IMGS_WIDTH TRACK_IMGS_HEIGHT TRACK_IMGS_MOTIONSIZE</li>
+        </ul>
+        <p></p>
+        For instance, the following script may be called to move a PTZ netcam (Tested with VIVOTEK PZ81X1):
+<pre><code>#!/bin/bash
+
+LOCKFILE=/tmp/track_generic_move_netcam.lock
+if [ -e "$LOCKFILE" ]; then  # Trick to avoid flooding
+  exit 0                     # the netcam of multiple
+fi                           # moving commands.
+
+function movecam() {
+  touch $LOCKFILE
+  curl "http://10.42.0.96/cgi-bin/camctrl/camctrl.cgi?move=$1"
```

Comment:
While it is sample that shouldn't really be cut/paste run, we may want to indicate `http://youripaddress/cgi...` rather than this specific ip.